### PR TITLE
Set terminal default profile

### LINKED ISSUE #1: update profile on new window
the terminal profile doesn't match the os profile in this scenario :

**given :**

-  terminal default profile is dark
-  os color mode is light

**when :**

1. quit terminal
1. open terminal

**then :**

terminal profile is dark. it should be light

### PR DESCRIPTION
**Pull request by e0da**
Add a `setTerminalDefaultProfile` function so that you can programatically set the default profile used for new terminals. Can be leveraged to fix https://github.com/patrik-csak/auto-terminal-profile/issues/1 by providing an option in the app to update the default app on update-profile, which will cause newly created terminal windows to utilize the preferred profile.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
-import {runAppleScript} from 'run-applescript';
+import { runAppleScript } from "run-applescript";
 
 /**
  * @param {string} profile
  * @return {Promise<void>}
  */
 export async function setTerminalProfile(profile) {
-	await runAppleScript(`tell application "Terminal"
+  await runAppleScript(`tell application "Terminal"
```
Comment (e0da) [inline index.js]: The formatting changes were applied automatically by my editor via Prettier. I figured it wouldn't hurt to leave them in.

**Comment by patrik-csak** [comment]
thanks so much for the contribution @e0da ! i think this is a great way to solve patrik-csak/auto-terminal-profile#1

i just pushed some changes. if you approve, i'll merge this and release. do you plan to work on patrik-csak/auto-terminal-profile#1?

File: index.js (hunk 1 above)
Comment (patrik-csak) [reply index.js]: this project uses [XO](https://github.com/xojs/xo)

https://github.com/patrik-csak/terminal-profile/blob/54dd4d4d38667b2cbd4ae725747a78752ee27b7d/package.json#L18

**Comment by nqkdev** [comment]
@patrik-csak can you merge and release?

**Comment by patrik-csak** [comment]
@nqkdev done!

**Comment by e0da** [comment]
Hey thank you so much for accepting this PR! Sorry I didn't get back to you sooner. I haven't taken any time to follow up on email and open source stuff lately.

I would also like to follow up on https://github.com/patrik-csak/auto-terminal-profile/issues/1 or in some way address the automatic behavior that I want from that, which is to have option(s) to also sync the default terminal. I hope to get to spend some time with it soon. You'll hear from me if I do. 😄

**Pull request merged**